Dockerfile

```
@@ -13,6 +13,7 @@ RUN deploy/update-version.sh version.yaml \
 
 FROM alpine:3.14.0 AS final
 RUN apk add --no-cache ca-certificates
+RUN apk add tzdata
```

Same comment as in https://github.com/iver-wharf/wharf-provider-gitlab/pull/20#discussion_r690166840

> Remember to run `apk` without cache or remove the cache when done to minimize the docker image sizes. Could reuse the above `RUN` statement as well to minimize the docker layers as well, like so:
>
> ```suggestion
> RUN apk add --no-cache ca-certificates tzdata
> ```

---

Nice touch! Taken.